### PR TITLE
Update scl-extension XSD file

### PR DESCRIPTION
**Pull request by AliouDIAITE**

### CODE REVIEW

File: scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd [hunk 1]
```diff
@@ -222,19 +223,20 @@ SPDX-License-Identifier: Apache-2.0
...
+                Define the specific Compas attributes linked to the Bay. The Main and Second Labels are label of the bay specific to each user.
+                MainLabel and Version identify uniquely the instanciated Bay as the UUID
                 The BayCodif is specific to each user to identify the bay kind.
             </xs:documentation>
         </xs:annotation>
         <xs:attribute name="UUID" type="xs:string" use="optional"/>
         <xs:attribute name="Indice" type="xs:string" use="optional"/>
         <xs:attribute name="Version" type="xs:string" use="optional"/>
-        <xs:attribute name="MainShortLabel" type="xs:string" use="optional"/>
+        <xs:attribute name="MainShortLabel" type="xs:string" use="required"/>
```
Comment (Sander3003) [inline scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd:234]: What is ment by required? Should every CoMPAS project has this attribute? Or only if tCompasBay is used?

Comment (AliouDIAITE) [reply scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd:234]: It is required only when Private tCompasBay is used (present in file).

Comment (Sander3003) [reply scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd:234]: It is solved: it only applies if tCompasBay is used.

**Review by Sander3003 — APPROVED** [review]
Looks good

**Pull request merged**